topological_navigation_msgs/package.xml

```diff
@@ -1,27 +1,29 @@
 <?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
 <package format="3">
   <name>topological_navigation_msgs</name>
-  <version>2.4.0</version>
+  <version>3.4.0</version>
   <description>The topological_navigation_msgs package</description>
 
   <maintainer email="abinch@sagarobotics.com">Adam Binch</maintainer>
   <maintainer email="jpfentanes@sagarobotics.com">Jaime Pulido Fentanes</maintainer>
-
   <license>MIT</license>
 
-  <author>Adam Binch</author>
-  <author>Jaime Pulido Fentanes</author>
```

Comment:
Have you removed the authors on purpose? I’d leave them in, but add yourself, @Iranaphor

Reply:
Same for the maintainers

Reply:
There was a lot of copying and pasting files so i must have missed some. Now it is building, I'll go through and add back in the original authors and maintainers (+ myself).